### DISCUSSION TITLE
신약개발을 위한 실전 약동학 I 동영상 강의 관련 문의입니다.

### DISCUSSION THREAD

**Discussion by HSLWi** [pipetcpt/pharmapk#10] [Q&A]
안녕하세요, 교수님

저는 고려대학교 생명과학대학에서 관련 전공을 하고있는 박사과정 학생입니다.
약동학과 관련하여 처음 공부를 하고자 검색 중에 신약개발을 위한 실전 약동학 I 관련 공개 서적을 알게되었습니다.
PIPET LMS ( http://pipetlms.or.kr/ )에 동영상 강의를 무료 공개중이시라는 내용을 보고 방문하였으나 강의를 볼 수가 없었습니다.
혹시 강의 시청이 가능할 수 있을지 여쭤보려고 여기에 방문드렸습니다.
괜찮으시다면 동영상 강의를 받아 볼 수 있을까요?

감사합니다.
좋은 하루 되세요.

**Comment by shanmdphd** [comment]
네 PIPETLMS 세팅 검토해 보겠습니다. 알려주셔서 감사드립니다.
